src/internal/components/chart-plot/index.tsx

```diff
@@ -85,11 +85,12 @@ function ChartPlot(
     children,
     onClick,
     onKeyDown,
-    onFocus,
-    onBlur,
     focusOffset = DEFAULT_PLOT_FOCUS_OFFSET,
     activeElementFocusOffset = DEFAULT_ELEMENT_FOCUS_OFFSET,
-    ...restProps
+    onMouseMove,
+    onMouseOut,
+    onApplicationBlur,
```

Comment:
Renamed `onFocus` and `onBlur` to `onApplicationFocus` and `onApplicationBlur` because these functions are called when focusing and blurring the application (inner highlighting), not the wrapping SVG plot.